bundles/com.espressif.idf.ui/plugin.xml

```diff
@@ -297,6 +297,23 @@
 				</with>
          	</visibleWhen>
          </command>
+         <command
+               commandId="com.espressif.idf.ui.installidfcomponents"
+               icon="icons/ESP-IDF Install Tools.png"
+               label="%command.label.6"
+               style="push">
+            <visibleWhen>
+               <with
+                     variable="activeMenuSelection">
+                  <iterate
+                        ifEmpty="false">
+                     <adapt
+                           type="org.eclipse.core.resources.IResource">
+                     </adapt>
+                  </iterate>
```

Comment:
We could also add an idf nature check just to make sure this menu is visible only for idf projects

Reply:
Done

Reply:
Done